### PR TITLE
TDR-3158 - Various usability/accessibility improvements on upload page

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

**Review by thanhz — COMMENTED** [review]
Looks good to me,  can you double check if this occurs for you? the word 'files' is repeated in the success message
Screenshot from 2023-05-22 12-35-49

File: app/views/standard/upload.scala.html [hunk 1]
```diff
@@ -48,60 +50,53 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
...
+                                <label for="file-selection" class="govuk-button govuk-button--secondary drag-and-drop__button">
+                                    Choose folder
+                                </label>
+                            </div>
                         </div>
-                    </div>
-                    <div>
+                        <div id="success-and-removal-message-container" class="govuk-summary-list govuk-file-upload" hidden="true">
+                            <div class="js-drag-and-drop-selected drag-and-drop__selected" id="item-selection-success-container">
+                                <p id="success-message-text" aria-live="assertive" aria-atomic="true" class="govuk-!-margin-bottom-3 govuk-!-margin-top-0 drag-and-drop__selected__description">The folder <strong id="files-selected-folder-name" class="folder-name"></strong> (containing <span class="folder-size"></span> files) has been selected.</p>
```
Comment (thanhz) [inline app/views/standard/upload.scala.html:76]: Can remove the word "files" from here (containing <span class="folder-size"></span> files) as it is duplicated in the success message

**Review by thanhz — COMMENTED** [review]
Looks good to me, just a small thing

File: app/views/standard/upload.scala.html (hunk 1 above)
Comment (vimleshtna) [reply app/views/standard/upload.scala.html]: Good catch. Removed it, thanks.

**Review by thanhz — APPROVED** [review]
Thanks, looks good

**Pull request merged**